app/generic/proof/facades/acapy_proof.py

```diff
@@ -0,0 +1,88 @@
+from abc import ABC, abstractmethod
+from typing import Optional
+
+from aries_cloudcontroller import AcaPyClient, IndyProofRequest
+from aries_cloudcontroller.model.indy_pres_spec import IndyPresSpec
+
+from app.generic.proof.models import PresentationExchange
+
+
+class Proof(ABC):
```

Comment:
I'm thinking what the best naming would be for proof. For issue credential we called it `issuer` (which is a role), while `proof` is more a statement. 

We should come up with something consistent for both

- issuer
- prover

or 

- credentials
- proofs

or

- issuer
- verifier

what do you think?

Reply:
hmm, that's a very good point actually. lt me have a think about it.

Reply:
went for issuer/verifier combo